apps/accounts/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from apps.users.models import UserProfile
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = (
+            'username', 'display_name', 'email',
+            'bio', 'deleted', 'display_collections',
+            'display_collections_fav', 'homepage',
+            'location', 'notes', 'notifycompat',
+            'notifyevents', 'occupation', 'picture_type',
```

Comment:
Although `last_login_* stuff I can see being useful to a user and something that will be shown on the profile page: https://addons.mozilla.org/en-US/firefox/user/amckay/